src/ctia/http/routes/actor.clj

```diff
@@ -12,7 +12,7 @@
     (POST "/" []
       :return StoredActor
       :body [actor NewActor {:description "a new Actor"}]
-      :header-params [api_key :- s/Str]
+      :header-params [api_key :- (s/maybe s/Str)]
```

Comment:
The generative tests (using allow-all auth now) helped catch this bug in a few places.